examples/deal.II/bps.h

```diff
@@ -202,8 +203,26 @@ class OperatorCeed : public OperatorBase<Number>
     // 2) create shape functions -> "ShapeInfo"
     const unsigned int fe_degree  = fe.tensor_degree();
     const unsigned int n_q_points = quadrature.get_tensor_basis()[0].size();
-    CeedBasisCreateTensorH1Lagrange(
-      ceed, dim, n_components, fe_degree + 1, n_q_points, CEED_GAUSS, &sol_basis);
+    {
+      dealii::internal::MatrixFreeFunctions::ShapeInfo<double> shape_info;
+      shape_info.reinit(quadrature, fe, 0);
+      dealii::internal::MatrixFreeFunctions::UnivariateShapeData<double> shape_data =
+        shape_info.get_shape_data();
+      std::vector<CeedScalar> q_ref_1d;
+      for (const auto q : quadrature.get_tensor_basis()[0].get_points())
+        q_ref_1d.push_back(q(0));
```

Comment:
This is fine, but, the same 1D data should be in: https://www.dealii.org/developer/doxygen/deal.II/structinternal_1_1MatrixFreeFunctions_1_1UnivariateShapeData.html#aeb5b85f1120f2957d39b51e2d348aedb